src/core/constants.js

```diff
@@ -42,6 +42,22 @@ export const validAddonTypes = [
   EXTENSION_TYPE,
   THEME_TYPE,
 ];
+// Mapping of the add-on types we show in URLs, etc. and what they map
+// to in the API (and how they're represented internally in the app).
+//
+// Examples:
+// * '/extensions/' -> 'extension'
+// * '/themes/' -> 'persona'
+export const API_ADDON_TYPES_MAPPING = {
+  extensions: EXTENSION_TYPE,
+  themes: API_THEME_TYPE,
```

Comment:
I agree, but:

![it-was-like-that-when-i-got-here](https://cloud.githubusercontent.com/assets/90871/21905009/c26749b4-d8fd-11e6-9502-31115f205983.jpg)

I'll see about getting rid of `API_THEME_TYPE` and just making it `THEME_TYPE`.